src/day8/re_frame_10x/components/re_com.cljs

```diff
@@ -617,4 +623,225 @@
   [& args]
   (clojure.string/join " " args))
 
-
+(defn loggable-args
+  "Return a version of args which is stripped of uninteresting values, suitable for logging."
+  [args]
+  (if (map? args)
+    (->> ;; Remove args already represented in component hierarchy
+      (dissoc args :src :child :children :panel-1 :panel-2 :debug-as)
+      ;; Remove args with nil value
+      (remove (comp nil? second))
+      (into {}))
+    args))
+
+(defn short-component-name
```

Comment:
You can remove all :data-rc and :data-rc-src attributes which should enable deleting loggable-args and short-component-name. These are debug features for re-com that are not relevant in 10x at the present time for the components we copy across using the current structure.